voxblox/include/voxblox/io/layer_io_inl.h

```diff
@@ -0,0 +1,248 @@
+#ifndef VOXBLOX_CORE_IO_LAYER_IO_INL_H_
+#define VOXBLOX_CORE_IO_LAYER_IO_INL_H_
+
+#include <fstream>
+
+#include "./Block.pb.h"
```

Comment:
I am really not a fan of relative file paths in includes, in this case couldn't this just be "Block.pb.h" anyway?

Reply:
I think this specifies not to search anywhere else for this file (i.e., not the whole path) but rather look for it in this specific include path, since it's a locally-generated file? @mfehr I have no idea what the conventions around protobufs are. Help?